ArduPlane/quadplane.cpp

```diff
@@ -1754,6 +1753,11 @@ void QuadPlane::update_throttle_hover()
         return;
     }
 
+    // do not update if quadplane forward motor is running (wing may be generating lift)
```

Comment:
@tridge @IamPete1 This is supposed to check whether there is forward thrust (in a non-tailsitter) and return early to prevent learning incorrect hover thrust

Reply:
makes sense, good idea